pkg/controllers/provisioning/suite_test.go

```diff
@@ -113,6 +114,17 @@ var _ = Describe("Provisioning", func() {
 		Expect(len(nodes.Items)).To(Equal(1))
 		ExpectScheduled(ctx, env.Client, pod)
 	})
+	It("should provision single pod if no other pod is received within the batch idle duration", func() {
+		ExpectApplied(ctx, env.Client, test.NodePool())
```

Comment:
Should we also validate that the same pod given multiple times doesn't extend our timeout? We probably need to customize the batch idle duration to be able to test this